gnovm/pkg/gnomod/pkg.go

```diff
@@ -111,17 +114,15 @@ func ListPkgs(root string) (PkgList, error) {
 			return fmt.Errorf("validate: %w", err)
 		}
 
+		imports, err := gnoimports.PackageImports(path)
+		_ = err // get valid imports while ignoring bad/partial files
+		imports = filterImports(gnoMod.Module.Mod.Path, imports)
```

Comment:
You can probably use `slices.DeleteFunc` or something similar

Reply:
6aea21eb83212d5620e5b9acfde96bdec5e5413d